src/__tests__/routing.test.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import { mountWithContext } from '../testUtils';
+import { cleanup } from 'react-testing-library';
 import App from '../App';
+import { mockUnauthenticated } from '../mocks/localStorageMock';
+
+afterEach(cleanup);
```

Comment:
Huh, I didn't know you could run `afterEach` outside of a `describe` block

Reply:
Frankly, I'm not actually sure it's doing anything, but it also doesn't hurt?